### ISSUE TITLE
Support Lark

### ISSUE THREAD

**Issue by jeremywood** [bdkjones/MuzzleApp#73]
Lark is an up and coming collaboration tool, and it would be great if Muzzle worked with it!

https://www.larksuite.com

**Comment by bdkjones** [comment]
Hi Jeremy,

Are you sure it doesn’t currently?

-Bryan

**Comment by jeremywood** [comment]
Yeah, unfortunately it doesn't work. Want to make sure I'm testing it correctly:

1.) Start screen-sharing in Lark.
2.) Check DND in the top of the Notification Center.

**Comment by bdkjones** [comment]
And you're sure you're **sharing the screen** and not just on a video call? I get a lot of people who misunderstand exactly when Muzzle kicks in.

**Comment by jeremywood** [comment]
Yeah, definitely sharing the screen.
